pkg/account/handler.go

```diff
@@ -0,0 +1,202 @@
+package account
+
+import (
+	"container/list"
+	"crypto/rand"
+	"net/http"
+	"regexp"
+
+	"github.com/labstack/echo/v4"
+)
+
+var account_list = list.New()
+
+// type AcountType string
+
+type AccountHandler struct {
+}
+
+// @Summary create and account based on email(as id), type, name and password
+// @Description will check primarykey other, then add to account_list if possible
+// @Tags Account
+// @Produce json
+// @Param email formData string true "user e-mail"
```

Comment:
Please do not use formData, we will use json for both direction of data transmission.